tests/e2e/backup_restore_suite_test.go

```diff
@@ -91,29 +165,17 @@ var _ = Describe("AWS backup restore tests", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
-	type BackupRestoreCase struct {
-		ApplicationTemplate  string
-		ApplicationNamespace string
-		Name                 string
-		BackupRestoreType    BackupRestoreType
-		PreBackupVerify      VerificationFunction
-		PostRestoreVerify    VerificationFunction
-		MaxK8SVersion        *K8sVersion
-		MinK8SVersion        *K8sVersion
-	}
-
-	updateLastInstallingNamespace := func(namespace string) {
-		lastInstallingApplicationNamespace = namespace
+	updateLastInstallTime := func() {
 		lastInstallTime = time.Now()
 	}
 
 	DescribeTable("backup and restore applications",
 		func(brCase BackupRestoreCase, expectedErr error) {
-			if notVersionTarget, reason := NotServerVersionTarget(brCase.MinK8SVersion, brCase.MaxK8SVersion); notVersionTarget {
-				Skip(reason)
+			// Data Mover is only supported on aws and azure.
```

Comment:
afaik we now have GCP support

Reply:
Also follow up.

Reply:
Have to double check if for GCP we use different secrets etc.

Reply:
Added GCP to case. Updating comment.
```suggestion
			// Data Mover is only supported on aws, azure, and gcp.
```